Tools/autotest/arducopter.py

```diff
@@ -9868,6 +9868,23 @@ def GPSForYaw(self):
         if ex is not None:
             raise ex
 
+    def SMART_RTL_EnterLeave(self):
+        '''check SmartRTL behaviour when entering/leaving'''
+        # we had a bug where we would consume points when re-entering smartrtl
```

Comment:
Thanks for this.  Not that it matters too much because it's all in the past but the bug was that points would be lost if SmartRTL mode was exited before reaching home

Would it really "Cause a panic"?  A panicking pilot perhaps but not a code execution panic I don't think.

Reply:
This bug isn't in the past.  This test is being merged *disabled*.  It still causes SITL to panic.

I've pulled it out of https://github.com/ArduPilot/ardupilot/pull/17419 so the test doesn't bitrot (which it had).  I'll be rebasing the fix PR on top of master soon (it's a non-trivial rebase as it conflicts badly with the recent consumption-of-points fix we merged.

Reply:
Ah right, sorry.  I assumed this was related to my recent fix but no, this is for that long standing bug.  OK thanks!